tool/requote.js

```diff
@@ -21,6 +21,15 @@ const StreamUtils = require('../lib/stream-utils');
 const ENTITY_MATCH_REGEX = /^([A-Z].*)_[0-9]+$/;
 const NUMBER_MATCH_REGEX = /^([0-9|\u0660-\u0669]+)$/;
 
+function do_replace_numbers(token, requote_numbers) {
+    // check if token is:
+    // 1) an Arabic or English number
+    // 2) ignore digit 0 and 1 since it is sometimes used in program to represent
+    // singularities (e.g. a, an, one, ...) but is not present in the snetence
```

Comment:
"Small numbers" have been expanded to the range [-12, 12] recently.

Reply:
But the numbers which are not "0" or "1" are present in the sentence and requotable, right?